### PR TITLE
Devops

### PR DESCRIPTION
**Pull request by termlen0**
##### SUMMARY
A new workshop candidate. This workshop's objective is uplevel the workshop experience to modern DevOps experiences including tools and practices.

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME

- exercises
- docs
- decks
- provisioner
- demos

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by termlen0** [comment]
cc: @rhcreynold

File: provisioner/provision_lab.yml [hunk 1]
```diff
@@ -157,6 +157,21 @@
...
 
+- name: CONFIGURE WORKSHOP WEBSERVERS FOR DEVOPS
+  hosts: lab_hosts
+  become: yes
+  gather_facts: no
+  roles:
+    - { role: devops, when: workshop_type == "devops" }
+
+
+- name: CONFIGURE F5 WORKSHOP WEBSERVERS host1 and host2
```
Comment (IPvSean) [inline provisioner/provision_lab.yml]: why is this F5 thing added for the `devops` workshop type?

File: provisioner/roles/control_node/tasks/devops.yml [hunk 2]
```diff
@@ -0,0 +1,59 @@
+# - name: Clone lightbulb
```
Comment (IPvSean) [inline provisioner/roles/control_node/tasks/devops.yml]: why is this commented out?

File: provisioner/roles/control_node/templates/ansible.cfg.j2 [hunk 3]
```diff
@@ -7,6 +7,8 @@ host_key_checking = False
 retry_files_enabled = False
 {% if (workshop_type == 'rhel') or (workshop_type == 'security') %}
 inventory = /home/{{username}}/lab_inventory/hosts
+{% elif workshop_type == 'devops' %}
```
Comment (IPvSean) [inline provisioner/roles/control_node/templates/ansible.cfg.j2]: curious on your opinion if we should start moving this all to `workshop` and simplify these templates down :) . or if the name is helpful here

File: provisioner/roles/devops/tasks/main.yaml [hunk 4]
```diff
@@ -0,0 +1,27 @@
+- name: Install EPEL
```
Comment (IPvSean) [inline provisioner/roles/devops/tasks/main.yaml]: EPEL is already installed on control node?

File: provisioner/roles/manage_ec2_instances/tasks/resources.yml [hunk 5]
```diff
@@ -85,9 +85,15 @@
         from_port: 8080
         cidr_ip: 0.0.0.0/0
       - proto: tcp
-        to_port: 8088
-        from_port: 8088
+        to_port: 8081
```
Comment (IPvSean) [inline provisioner/roles/manage_ec2_instances/tasks/resources.yml]: what is port 8081 one for, i wonder if we should add descriptions to each line here for others to help work on these, or move this entirely into vars

**Comment by IPvSean** [comment]
@termlen0 can you rebase and address the comments, thanks!

**Comment by IPvSean** [comment]
still waiting on comments to be addressed

**Comment by IPvSean** [comment]
bump, @termlen0 can you look at comments?

**Comment by termlen0** [comment]
Closing PR to address bug fixes. Will resubmit.

**Pull request closed without merging**